common/network-common/src/main/java/org/apache/spark/network/sasl/SparkSaslClient.java

```diff
@@ -50,6 +52,9 @@
   private final String expectedQop;
   private SaslClient saslClient;
 
+  private boolean isAesEnabled;
```

Comment:
Is it possible to remove isAesEnabled and use sparkSaslAES to check?

if(sparkSaslAES != null) {
 }